Resources/string.csv

```diff
@@ -85,148 +85,148 @@ TeamImpostor,Team Impostors,インポスター陣営,内鬼阵营,狼人陣營
 NeutralInfo,A solitary teams that does not belong to any other teams,他のどの陣営にも属さない孤高の陣営,不属于其他阵营的独立阵营,你屬於一個獨立陣營且不受其他陣營或職業的獲勝條件引響
 
 # 特殊インポスター役職
-BountyHunterInfo,Hunt your bounty down,標的を確実に仕留めよう,猎杀你的赏金目标,將你的目標拿下
+BountyHunterInfo,Hunt your bounty down,標的を確実に仕留めよう,猎杀你的目标,將你的目標拿下
 MareInfo,Kill in the dark,闇の世界で暗殺せよ,在黑暗中作案,在黑暗中獵殺
-FireWorksInfo,Bloom at the last,最後に一華咲かせよう,一起来看最后的烟花吧,在最後時刻來看煙火秀吧
-SerialKillerInfo,You crave blood,殺人衝動が抑えられない,你渴望鲜血，抑制不住你的冲动,你無法抑制你的殺人衝動
-ShapeMasterInfo,Shapeshift and confuse the Crewmates,変身して敵を混乱させよう,变幻成其他人来迷惑敌人,變換外表來迷惑他人
+FireWorksInfo,Bloom at the last,最後に一華咲かせよう,快来从炮筒里看烟花吧,在最後時刻來看煙火秀吧
+SerialKillerInfo,You crave blood,殺人衝動が抑えられない,你渴望鲜血,你無法抑制你的殺人衝動
+ShapeMasterInfo,Shapeshift and confuse the Crewmates,変身して敵を混乱させよう,变幻外貌迷惑他人,變換外表來迷惑他人
 VampireInfo,Bite the Crewmates,クルーを噛んで全滅させよう,吸干他人的血,使用你的牙齒吸乾所有人的血吧
 WarlockInfo,Curse the Crewmates,敵を呪い殺そう,给你的敌人下咒,詛咒並殺光你的敵人
 WitchInfo,Spell the Crewmates,敵に魔術をかけよう,诅咒你的敌人,使用你的魔法來讓你的敵人死亡
 MafiaInfo,Help the Impostors to kill everyone,インポスターの援助をしよう,帮助其他内鬼杀掉所有人,幫助狼人殺死所有人
 BeforeMafiaInfo,You can't kill yet,今はキルをすることができない,潜伏以等候时机,潛伏以等待時機
 AfterMafiaInfo,It's time to kill,サボを活用して皆殺しにしよう,杀光所有人,殺光所有人吧
-PuppeteerInfo,Force the Crewmate to kill,敵を操って皆殺しにしよう,操纵敌人，杀光船员,操控對象並殺光所有船員
-TimeThiefInfo,Kill the Crewmates to steal discussion time,敵を殺し、会議時間を奪おう,杀死所有船员并偷取时间,殺光你的敵人並將會議時間縮短
-SniperInfo,Shoot the Crewmates in the distance,狙い撃つぜ,瞄准敌人，射击！,讓你的敵人在你的狙擊下死亡
+PuppeteerInfo,Force the Crewmate to kill,敵を操って皆殺しにしよう,操纵别人，杀光船员,操控對象並殺光所有船員
+TimeThiefInfo,Kill the Crewmates to steal discussion time,敵を殺し、会議時間を奪おう,Too young too simple,殺光你的敵人並將會議時間縮短
```

Comment:
`Too young too simple`
It's not translated here, is that okay?

Reply:
It is translated. It is different from the English text, right?
This is a meme popular in China, and we use this to express something similar to 'time was stolen'.
We'll consider to change it if you don't like that. Thank you.

Reply:
I'm fine with either.
Only here is English, so I felt a sense of incongruity.

Reply:
I see. Thank you very much.